src/silo/query_engine/actions/aggregated.cpp

```diff
@@ -0,0 +1,304 @@
+#include "silo/query_engine/actions/aggregated.h"
+
+#include <algorithm>
+#include <chrono>
+#include <functional>
+#include <random>
+#include <utility>
+#include <vector>
+
+#include <tbb/enumerable_thread_specific.h>
+#include <tbb/parallel_for_each.h>
+#include <boost/algorithm/string/join.hpp>
+#include <nlohmann/json.hpp>
+
+#include "silo/database.h"
+#include "silo/query_engine/operator_result.h"
+#include "silo/query_engine/query_parse_exception.h"
+#include "silo/query_engine/query_result.h"
+#include "silo/storage/metadata_store.h"
+
+namespace silo::query_engine::actions {
+
+struct Tuple {
```

Comment:
```suggestion
struct RowValues {
```
or 
```suggestion
struct RowValuesOfInterest {
```
?
The second one is a bit lengthy, but makes the purpose clear.

Reply:
I want tuple. This is standard